tests/capture/input_generators/test_attack_point_iterator.py

```diff
@@ -293,7 +358,8 @@ def test_attack_point_iterator_repeat_infinite_minus_two():
         "configuration": {
             "operation": "constants",
             "name": "key",
-            "values": [1, 2]
+            "length": 2,
+            "values": [[1, 2]]
```

Comment:
This changes the unit-test. Before we were iterating
1
2
1
2
1
2
...

Now we are iterating always the same value.
[1,2]
[1,2]
[1,2]
...

Luckily there is another test which is covering that.

Reply:
Is this not good yet?

Reply:
Ah I have not resolved yet.